src/pages/MetricDetail.svelte

```diff
@@ -75,7 +77,12 @@
   </p>
   <table class="metrics-table">
     <tr>
-      <td>Relevant Bugs</td>
+      <td>
+        Relevant Bugs
+        <ToolTip top={true} tip={getHelpMessage('bugs')}>
```

Comment:
It probably makes sense to create a small component for this (in another project, I called a similar thing `HelpHoverable`)